examples/esp32spi_secrets.py

```diff
@@ -1,7 +1,7 @@
 # This file is where you keep secret settings, passwords, and tokens!
```

Comment:
should this file be name secrets.py  -- or keep it here as esp32spi_secrets.py to distinguish from other examples but user should rename to secrets.py